gui/src/main/java/DiceGameGUI.java

```diff
@@ -284,7 +282,15 @@ public void actionPerformed(java.awt.event.ActionEvent evt) {
 
     private void btnResetActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_btnResetActionPerformed
         // TODO add your handling code here:
-        thisGame = DiceGame.create();
+        String name = lblPlayerName.getText();
```

Comment:
@hehe876 probably not the best way for me to do this. The UI shouldn't be used to save state. But I'm not sure if I want to create a separate member variable just for this. What do you think?